docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -47,6 +47,24 @@ that if a specific machine image is specified, it has to match the Kubernetes ve
 `KubeadmControlPlane` spec. In order to only trigger a single upgrade, the new `MachineTemplate` should be created first
 and then both the `Version` and `InfrastructureTemplate` should be modified in a single transaction.
 
+#### How to trigger a rollout irrespective of spec changes
```

Comment:
What about?
```suggestion
#### How to schedule machine rollout
```
I think that schedule could make easier to understand upgradeAfter for end users

Reply:
Done, I updated the title, that's more succinct indeed. I also added a paragraph to explicitly say that `UpgradeAfter` can't be used for "batching" changes, i.e. that it would not delay an upgrade that was triggered by modifying the spec.

Not sure if that's too much, but I wanted to be explicit as this was what we originally thought it could be used for until we read the code.